src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -1369,7 +1369,7 @@ private static bool IsEffectivelyPrivate(Symbol symbol)
         internal bool IsAccessible(Symbol symbol, ref HashSet<DiagnosticInfo> useSiteDiagnostics, TypeSymbol accessThroughType = null, ConsList<Symbol> basesBeingResolved = null)
         {
             bool failedThroughTypeCheck;
-            return IsAccessible(symbol, accessThroughType, out failedThroughTypeCheck, ref useSiteDiagnostics, basesBeingResolved);
+            return IsAccessible(symbol, accessThroughType, out _, ref useSiteDiagnostics, basesBeingResolved);
```

Comment:
unfortunate that we left the 'bool failedThroughTypeCheck' around.

Reply:
Yes, I will try to fix the code fixer here.

Reply:
👍

Reply:
Fixed with https://github.com/dotnet/roslyn/pull/30777/commits/97424a343f354e6f3aed1f0c6000a9cda48c4953